lib/plugins/babelRuntimePathResolver.js

```diff
@@ -2,11 +2,12 @@
 // in OBT's node_modules directory and not the module's
 'use strict';
 
-const path = require('path');
 const falafel = require('falafel');
 
 const prefix = /babel-runtime/;
-const rootDirectory = path.resolve(__dirname, '../../node_modules');
+
+// HACK: Can't require.resolve babel-runtime so use babel-core instead
+const rootDirectory = require.resolve('babel-core').replace(/(.*node_modules).*/, '$1');
```

Comment:
unless I'm mistaken, yes it's already broken in this way. but this brokenness is hidden by the fact that in *most* cases everything will be findable from that same root. the problem comes if you've got something else going on, like maybe you've npm link'd to your own development version of origami-build-tools.

Another edge case would be if you depended on a module foo v1, but origami-build-tools depends on foo v2 for some internal thing (unbeknownst to you) so you end up getting the wrong API and you might get very confusing bugs. So I think we need to specifically rewrite *only* when the requested module is `babel-runtime/**`.

I'll try reworking my one...